### PR TITLE
feat(ui): log unhandled exceptions

### PR DESCRIPTION
**Pull request by geigerzaehler**
As part of #1187 we show notifications for all unhandled exceptions.

### CODE REVIEW

**Review by xla — APPROVED** [review]
🐤 🍤 🏂 🕜

File: ui/src/error.ts [hunk 1]
```diff
@@ -16,9 +16,11 @@ export interface Error {
 }
 
 export enum Code {
-  SessionFetchFailure = "SessionFetchFailure",
   ProjectRequestFailure = "ProjectRequestFailure",
+  SessionFetchFailure = "SessionFetchFailure",
   UnexpectedProxyExit = "UnexpectedProxyExit",
```
Comment (FintanH) [inline ui/src/error.ts:21]: Do you want to do the rename you mentioned in the docs PR now?

**Review by FintanH — APPROVED** [review]
Nice!

File: ui/src/error.ts (hunk 1 above)
Comment (geigerzaehler) [reply ui/src/error.ts:21]: Will do separately

**Pull request merged**